client/dashboard/sites/backups/backup-details.tsx

```diff
@@ -92,6 +94,15 @@ export function BackupDetails( { backup, site }: { backup: ActivityLogEntry; sit
 							</Text>
 						) }
 					</HStack>
+					<HStack justify="flex-start" style={ { flexWrap: 'wrap' } }>
+						{ backup.streams ? (
+							backup.streams.map( ( item, index ) => (
+								<ImagePreview key={ index } item={ item } multipleImages />
+							) )
+						) : (
+							<ImagePreview item={ backup } />
+						) }
+					</HStack>
```

Comment:
I think we can improve how it looks by switching from HStack to a Grid. This will also requires a change in the inline CSS of the ImagePreview component: https://github.com/Automattic/wp-calypso/pull/105634/files#r2335648388

| Before | After |
|---|---|
| <img width="2642" height="1050" alt="CleanShot 2025-09-10 at 01 00 29@2x" src="https://github.com/user-attachments/assets/c32ff044-34a9-4c4c-b0c1-ce89f402f610" /> | <img width="2642" height="1050" alt="CleanShot 2025-09-10 at 00 59 54@2x" src="https://github.com/user-attachments/assets/91712788-e429-44de-9707-6cf8c54bb6f0" /> |

```suggestion
					<Grid templateColumns="repeat(auto-fit, minmax(200px, 1fr))">
						{ backup.streams ? (
							backup.streams.map( ( item, index ) => (
								<ImagePreview key={ index } item={ item } multipleImages />
							) )
						) : (
							<ImagePreview item={ backup } />
						) }
					</Grid>
```

Reply:
That was a nice suggestion, thanks. I've applied it.